pkg/components/master.go

```diff
@@ -120,6 +133,17 @@ func (m *master) createInitScript() string {
 		fmt.Sprintf("/usr/bin/yt set //sys/@cluster_connection '%s'", string(clusterConnection)),
 		"/usr/bin/yt set //sys/controller_agents/config/operation_options/spec_template '{enable_partitioned_data_balancing=%false}' -r -f",
 		m.initAdminUser(),
+		m.initMedia(),
+	}
+
+	if m.cfgen.HasSsdBlobs() {
```

Comment:
1. since `@primary_medium` is an inheritable attribute, it'is better to set the medium on the directory, and not to hardcode all table paths.
2. I think it's better to add desired medium for operations_archive to ytsaurus spec explicitly